test/fixtures/scenarioManager.js

```diff
@@ -6,9 +6,10 @@ var config = require('./config').scenarios;
 function ScenarioManager(server) {
   if (!server) throw new Error('No server defined');
 
+  // NOTE: some large sets of test data can take several minutes to load
   this.client = new elasticsearch.Client({
     host: server,
-    requestTimeout: 300000
+    requestTimeout: 3000000
```

Comment:
Was the previous timeout of 5 minutes not enough? A 50 minute timeout doesn't seem reasonable.